api-put-bucket.go

```diff
@@ -71,6 +71,12 @@ func (c *Client) doMakeBucket(ctx context.Context, bucketName string, location s
 		reqMetadata.customHeader = headers
 	}
 
+	if panfsPath != "" {
+		headers := make(http.Header)
+		headers.Add("x-panfs-bucket-path", panfsPath)
+		reqMetadata.customHeader = headers
```

Comment:
This can overwrite `customHeader` set on line 71 if `objectLockEnabled` is set as well.